src/routes/index.tsx

```diff
@@ -23,14 +25,18 @@ export const ROUTES = {
     defaultTimeRangeWeekOffset: -3,
     defaultScale: Scale.PERCENT,
   }),
-  "/": HomeRoute,
 };
 
 export function Routes(): VNode {
   return (
     <Router>
-      {Object.entries(ROUTES).map(([path, Component]) => (
-        <Component path={path} />
+      {Object.entries(ROUTES).map(([path, { title, component: Component }]) => (
+        <div path={path}>
+          <Header path={path} routes={ROUTES} />
+          <Page title={title}>
+            <Component />
```

Comment:
just realized this messes up our query param stuff... `<Component>` needs to be the top level thing routed to